vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -563,14 +568,12 @@ type internal FsiToolWindow() as this =
 
         executeTextNoHistory interaction
 
-    let sendSelectionToFSI dbgBreak =
+    let sendSelectionToFSI selectLine dbgBreak =
```

Comment:
Could you pass an `FsiEditorSendAction` value instead? I'm not fond those function call with boolean literals. You can pattern matching on `FsiEditorSendAction` which is nice.